### PR TITLE
TEC-3992 Sharing Block Re-Skin

### PR DESCRIPTION
**Pull request by EugeneKyale**
This PR contains some styles for the Sharing block reskin. 

Artifact : https://www.loom.com/share/7df920c8f69f41d185d684b753e94b67
Ticket : https://theeventscalendar.atlassian.net/browse/TEC-3992

### CODE REVIEW

**Comment by EugeneKyale** [comment]
This PR is very dependent on this [one](https://github.com/the-events-calendar/the-events-calendar/pull/3611). As a result of this I'll fix the failing tests one the former is merged.

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss [hunk 1]
```diff
@@ -134,6 +134,41 @@
...
+			&:before {
+				color: var(--tec-color-link-accent);
+				content: '+';
+				margin-right: 3px;
+			}
+		}
+
+		img {
+			display: none;
+			margin-right: 18px;
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks-reskin.pcss:167]: if this is display none, we don't need this margin right?

**Review by paulmskim — CHANGES_REQUESTED** [review]
just one small thing and should be good to go

File: src/resources/postcss/tribe-events-single-blocks-reskin.pcss (hunk 1 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks-reskin.pcss:167]: ooh yeah, we don't need the margin. I've made the update.

**Review by paulmskim — APPROVED** [review]

**Pull request merged**